src/components/SpecialResolution/SpecialResolutionForm.vue

```diff
@@ -0,0 +1,463 @@
+                  <DatePickerShared
+                    ref="signatureDatePickerRef"
+                    title="Date Signed"
+                    :nudgeRight="40"
+                    :nudgeTop="85"
+                    :minDate="signatureDateMinStr"
+                    :maxDate="signatureDateMaxStr"
+                    :inputRules="signatureDateRules"
+                    @emitDateSync="onSigningDateSync"
+                  />
+                </v-col>
+              </v-row>
+            </v-card>
+
+        </section>
+      </v-card>
+    </v-card>
+  </v-card>
+</template>
+
+<script lang="ts">
+// this is a placceholder copied from AlterationSummary, Will add component when working on this page
+import { Component, Mixins, Watch } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+import { ActionBindingIF, HelpSectionIF, ResourceIF,
+  FormIF, PersonIF, EmptyPerson, CreateResolutionIF } from '@/interfaces/'
+import { DateMixin, SharedMixin, FilingTemplateMixin, CommonMixin } from '@/mixins/'
+import { HelpSection } from '@/components/common/'
+import { DatePicker as DatePickerShared } from '@bcrs-shared-components/date-picker/'
+
+@Component({
+  components: {
+    HelpSection,
+    DatePickerShared
+  }
+})
+export default class SpecialResolutionForm extends Mixins(
+  DateMixin
+
+) {
+  @Getter getResource!: ResourceIF
+  @Getter getBusinessFoundingDate!: Date
+
+  @Action setResolution!: ActionBindingIF
+  @Getter getCreateResolutionStep!: CreateResolutionIF
+  @Getter getComponentValidate!: boolean
+
+  // Refs
+  $refs!: {
+    resolutionDatePickerRef: DatePickerShared,
+    signatureDatePickerRef: DatePickerShared,
+    signingPersonFormRef: FormIF,
+  }
+
+  // Date properties
+  protected resolutionDateText: string = ''
+  protected signatureDateText: string = ''
+
+  protected MAX_RESOLUTION_TEXT_LENGTH: number = 1000
+  protected resolutionText: string = ''
+
+  protected signingPerson: PersonIF = null
+
+  get helpSection (): HelpSectionIF {
+    return this.getResource.changeData?.specialSpecialResolution?.helpSection
+  }
+
+  get getCreateResolutionResource (): any {
+    return this.getResource.changeData?.specialSpecialResolution?.sampleFormSection || null
+  }
+
+  get documentURL (): string {
+    const docUrl = sessionStorage.getItem('BASE_URL') +
+      this.getCreateResolutionResource?.downloadDocPath
+    return docUrl
+  }
+
+  get previewImageSource (): string {
+    // Note: the image file path did not resolve correctly when using the require function directly.  In order
+    // to get the image path resolving correctly, needed to get the image context first and use that to build
+    // the final image file path
+
+    if (this.isJestRunning || !this.getCreateResolutionResource?.previewImagePath) { return '' }
+    const images = require.context('@/assets/images/', false, /\.png$/)
+    return images('./' + this.getCreateResolutionResource?.previewImagePath)
+  }
```

Comment:
Why don't you just hard-code the path in the img tag [like this](https://github.com/bcgov/business-create-ui/blob/73cd0b9344031b50579ff7bb898551c8adf0df17/src/views/Incorporation/IncorporationAgreement.vue#L196)?